test/appium/views/chat_view.py

```diff
@@ -1015,6 +1019,10 @@ def invited_to_join_system_message(username, chat_name):
     def join_system_message(username):
         return '%s joined the group' % username
 
+    @staticmethod
+    def you_have_joined_system_message(chat_name, admin):
```

Comment:
Is it used?

Reply:
thanx. removed unused def you_have_joined_system_message(chat_name, admin) and def create_for_admin_system_message(chat_name) methods